docs/governance/on-chain-process.md

```diff
@@ -44,29 +42,23 @@ If the proposal has been approved, provided the proposal’s feasibility, the xE
 
 ### Publishing the xEIP on-chain
 
-You can create the xEIP on xExchange governance page (https://xexchange.com/governance), and the xEIP is then published on-chain (in the governance smart contract).
+The xEIP can be published for on-chain voting by calling the governance smart contract, and will then be visible on xExchange governance page (https://xexchange.com/governance).
 
-To create the xEIP, you would first write down all the information about the xEIP in a PDF: title, summary, and detailed description. Then, you would upload the PDF on IPFS (a reliable decentralized storage solution). This will make sure that all users will vote on a common, censorship resistant and non-editable version of the proposal.
+To create the xEIP, you would first write down all the information about the xEIP in a text file like a PDF: title, summary, and detailed description. Then, you would hash that text file using [SHA-1 hash function](https://en.wikipedia.org/wiki/SHA-1). This will make sure that all users will vote on a common, censorship resistant and non-editable version of the proposal.
```

Comment:
PDF is not a text file (can contain images, etc) and is binary (not directly readable).

How do you hash a PDF?

I am not sure the last sentence "This will make ..." is relevant as is now we don't post on IPFS anymore.

Reply:
I agree. PDF is removed. Also reformulated the "This will make ..." part.